README.md

```diff
@@ -1,19 +1,12 @@
-# di-ipv-cri-common-lambdas
+# di-ipv-cri-common-lambdas: DI IPV Credential Issuer Common Lambdas
 
-DI IPV Credential Issuer Common Lambdas
+This code runs inside its own Repo. This repository is the home for common CRI Lambdas supporting Cloud Formation infrastructure which is shared or sensitive. If you are making changes to this repo please update [RELEASE_NOTES](./RELEASE_NOTES.md) so that teams can check for changes before re-deploying.
```

Comment:
`This code runs inside its own Repo.` not sure this is clear I think it can be removed b'cos it's a little ambiguous. The rest looks fine